<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>501</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="6" column="5">
      <widget class="Clickable_QLabel" name="label_54">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>54</string>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="Clickable_QLabel" name="label_56">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>56</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="Clickable_QLabel" name="label_37">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>37</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="Clickable_QLabel" name="label_24">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>24</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="Clickable_QLabel" name="label_39">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>39</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="Clickable_QLabel" name="label_29">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>29</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="Clickable_QLabel" name="label_36">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>36</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="Clickable_QLabel" name="label_61">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>61</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="Clickable_QLabel" name="label_23">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>23</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="Clickable_QLabel" name="label_22">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>22</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="Clickable_QLabel" name="label_21">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>21</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="Clickable_QLabel" name="label_57">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>57</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="Clickable_QLabel" name="label_12">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>12</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="Clickable_QLabel" name="label_31">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>31</string>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="Clickable_QLabel" name="label_55">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>55</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="Clickable_QLabel" name="label_25">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>25</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="Clickable_QLabel" name="label_26">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>26</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="Clickable_QLabel" name="label_60">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>60</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="Clickable_QLabel" name="label_44">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>44</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="Clickable_QLabel" name="label_15">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="Clickable_QLabel" name="label_30">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="Clickable_QLabel" name="label_53">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>53</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="Clickable_QLabel" name="label_45">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>45</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="Clickable_QLabel" name="label_1">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="Clickable_QLabel" name="label_34">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>34</string>
       </property>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="Clickable_QLabel" name="label_63">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>63</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="Clickable_QLabel" name="label_35">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>35</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="Clickable_QLabel" name="label_5">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="Clickable_QLabel" name="label_16">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="Clickable_QLabel" name="label_40">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>40</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="Clickable_QLabel" name="label_27">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>27</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="Clickable_QLabel" name="label_13">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>13</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="Clickable_QLabel" name="label_62">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>62</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="Clickable_QLabel" name="label_7">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Clickable_QLabel" name="label_2">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="Clickable_QLabel" name="label_8">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="Clickable_QLabel" name="label_19">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>19</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="Clickable_QLabel" name="label_43">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>43</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="Clickable_QLabel" name="label_49">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>49</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="Clickable_QLabel" name="label_51">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>51</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="Clickable_QLabel" name="label_9">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="Clickable_QLabel" name="label_6">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="Clickable_QLabel" name="label_42">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>42</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="Clickable_QLabel" name="label_52">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>52</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="Clickable_QLabel" name="label_28">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>28</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="Clickable_QLabel" name="label_3">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="Clickable_QLabel" name="label_32">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>32</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="Clickable_QLabel" name="label_11">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>11</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="Clickable_QLabel" name="label_59">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>59</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="Clickable_QLabel" name="label_50">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>50</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="Clickable_QLabel" name="label_38">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>38</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="Clickable_QLabel" name="label_4">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="7" column="7">
      <widget class="Clickable_QLabel" name="label_64">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>64</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Clickable_QLabel" name="label_10">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="Clickable_QLabel" name="label_33">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>33</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Clickable_QLabel" name="label_18">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>18</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="Clickable_QLabel" name="label_14">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>14</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="Clickable_QLabel" name="label_17">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>17</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="Clickable_QLabel" name="label_58">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>58</string>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="Clickable_QLabel" name="label_48">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>48</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="Clickable_QLabel" name="label_47">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>47</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="Clickable_QLabel" name="label_46">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>46</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="Clickable_QLabel" name="label_20">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="Clickable_QLabel" name="label_41">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>41</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>          Difficulty</string>
    </property>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Amateur</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Novice</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Expert</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>New game</string>
    </property>
   </widget>
   <widget class="QLabel" name="sign">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sign</string>
    </property>
   </widget>
   <widget class="QLabel" name="pieceInTurn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>pieceInTurn</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Clickable_QLabel</class>
   <extends>QLabel</extends>
   <header>clickable_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
